secator/output_types/exploit.py

```
@@ -28,20 +30,14 @@ class Exploit(OutputType):
 	_duplicate: bool = field(default=False, repr=True, compare=False)
 	_related: list = field(default_factory=list, compare=False)
 
-	_table_fields = [
-		MATCHED_AT,
-		NAME,
-		ID,
-		EXTRA_DATA,
-		REFERENCE
-	]
+	_table_fields = [MATCHED_AT, NAME, ID, EXTRA_DATA, REFERENCE]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)